sdk/core/core-rest-pipeline/src/util/userAgentPlatform-react-native.mts

```diff
@@ -4,7 +4,8 @@
 /*
  * NOTE: When moving this file, please update "react-native" section in package.json.
  */
-const { Platform } = require("react-native"); // eslint-disable-line import/no-extraneous-dependencies, @typescript-eslint/no-require-imports
+/* @ts-ignore */
+const { Platform } = await import("react-native"); // eslint-disable-line import/no-extraneous-dependencies, @typescript-eslint/no-require-imports
```

Comment:
is there a way to not use @ts-ignore?

no-require-imports no longer applies?

Reply:
No, since this is using `import`, it is statically analyzed therefore it will throw an error if react-native isn't there.